support/Pipelines/Bbh/Inspiral.py

```diff
@@ -130,18 +167,25 @@ def inspiral_parameters(
         "IdFromEvolution": id_from_evolution,
         # Domain geometry
         "ExcisionRadiusA": (
-            id_domain_creator["ObjectA"]["InnerRadius"] * excision_radius_factor
+            id_domain_creator["ObjectA"]["InnerRadius"]
+            * excision_radius_factor_a
         ),
         "ExcisionRadiusB": (
-            id_domain_creator["ObjectB"]["InnerRadius"] * excision_radius_factor
+            id_domain_creator["ObjectB"]["InnerRadius"]
+            * excision_radius_factor_b
         ),
+        "ObjectOuterRadius": initial_separation / 2.5,
         "XCoordA": id_domain_creator["ObjectA"]["XCoord"],
         "XCoordB": id_domain_creator["ObjectB"]["XCoord"],
         "CenterOfMassOffset_y": id_domain_creator["CenterOfMassOffset"][0],
         "CenterOfMassOffset_z": id_domain_creator["CenterOfMassOffset"][1],
+        "EnvelopeRadius": 100.0 / 15.0 * initial_separation,
+        "OuterShellRadius": 600.0 / 15.0 * initial_separation,
```

Comment:
Add a comment here that SpEC chooses the outer radius based on a Newtonian estimate of the wave zone (See function AutoRmax in SpEC/Support/Perl/SpEC.pm) and we can port that over if needed. Also we may have to adjust the CCE extraction radii.